### ISSUE TITLE
Simple product management

### ISSUE THREAD

**Issue by Daste745** [nutrishare/nutrishare#27]
Allow to create/edit products, so we can start creating a database of nutrition facts and barcodes. We'd at least need name, manufacturer, nutrition facts and a barcode number to collect some basic data. It would be nice to organize products into categories from the beginning.

~~A barcode scanner would be nice to have, but I'm not sure how that would work with our current (web) platform 🤔. This should probably be revisited in the future, as it would also be nice for retrieving products (when planning meals, etc.).~~
Use the native barcode scanner from Android/iOS. And allow to add a new product when the barcode doesn't match any existing product.

Importing data from an external service, like https://pl.openfoodfacts.org/ seems like a very compelling option, especially at the start, when we don't yet have *any* products. If someone scans a code that we don't have, we could request data from openfoodfacts and save it to our DB. We could even contribute back if that's possible 🤔.

Tracking:
- [x] #22 should be done before we start implementing actual features
- [x] #28
- [ ] #29
- [ ] #35
